docs/connectors/slack.md

```diff
@@ -19,34 +26,65 @@ connectors:
     # required
     token: "xoxb-abdcefghi-12345"
     # optional
-    user-token: "xoxp-abcdef-123" # default None. ** 
-    bot-name: "mybot" # default "opsdroid"
+    socket-mode: True # defaul false. if true app-token is required
+    app-token: "xapp-abdcfkje-12345" # socket-mode needs to be true
```

Comment:
Eugh slacks tokens are fun. Thinking about this the other day, I have a proposal. 

Why don't we not reuse the "token" key. This means that any config with `token:` in it would be invalid when used with the Events API and we can then catch this and print out to the user a nice set of instructions on how to update their slack app and how to switch to the events API.